doc/user/content/manage/monitor/datadog.md

```diff
@@ -0,0 +1,159 @@
+---
+title: "Datadog"
+description: "How to monitor the performance and overall health of your Materialize region using Datadog."
+menu:
+  main:
+    parent: "monitor"
+---
+
+This guide walks you through the steps required to monitor the performance and
+overall health of your Materialize region using [Datadog](https://www.datadoghq.com/).
+
+## Before you begin
+
+To make Materialize metadata available to Datadog, you must configure and run
+the following additional services:
+
+* A Prometheus SQL Exporter.
+* A Datadog Agent configured with an [OpenMetrics check](https://docs.datadoghq.com/integrations/openmetrics/).
+
+{{< note >}}
+In the future, we plan to support a native Datadog integration that
+continually reports metrics via the Datadog API {{% gh 17601 %}}.
+{{< /note >}}
+
+## Step 1. Set up a Prometheus SQL Exporter
+
+To export metrics from Materialize and expose them in a format that Datadog can
+consume, you need to configure and run a Prometheus SQL Exporter. This service
+will run SQL queries against Materialize at specified intervals, and export the
+resulting metrics to a Prometheus endpoint.
+
+We recommend using [`justwatchcom/sql_exporter`](https://github.com/justwatchcom/sql_exporter),
+which has been tried and tested in production environments.
+
+1. In the host that will run the Prometheus SQL Exporter, create a configuration
+   file (`config.yml`) to hold the Exporter configuration.
+
+   **Tip:** use [this sample `config.yml`](https://github.com/MaterializeInc/demos/blob/main/integrations/datadog/config.yaml)
+   as guidance to bootstrap your monitoring with some key Materialize metrics
+   and indicators.
+
+1. In the configuration file, define the connection to your Materialize region
+   under `connections` using the credentials provided in the [Materialize console](https://console.materialize.com/).
+
+   {{< note >}}
+   You must escape the special `@` character in `USER` for a successful
+   connection. Example: instead of `name@email.com`, use `name%40email.com`.
+   {{</ note >}}
+
+   _config.yml_
```

Comment:
The `dbt` guide uses the following indicator for the filename. Do we want to apply the same here?
```suggestion
   **Filename:** config.yml
```